src/components/Kv/KvIconButton.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+	<kv-button class="icon-btn"
```

Comment:
I like the theme idea here, but it doesn't allow us use our other existing button appearances easily without passing most of their styling in via CSS custom properties.

Another way to go about this would be to add 2 more slots to KvButton.vue. Or potentially "icon", and "icon-right" props . That would require rewriting KvButton some to not use the render function. 

I can imagine `<kv-button class="ghost" icon="info" icon-right="chevron">` or something like that.